libs/gretty/src/main/groovy/org/akhikhl/gretty/GrettyPlugin.groovy

```diff
@@ -280,17 +280,20 @@ class GrettyPlugin implements Plugin<Project> {
         }
       }
 
-      project.task('prepareInplaceWebAppClasses', group: 'gretty') {
+      project.tasks.register('prepareInplaceWebAppClasses') {
+        group = 'gretty'
         description = 'Compiles classes of this web-app and all overlays (if any)'
-        dependsOn project.tasks.classes
+        dependsOn project.tasks.named('classes')
+        // FIXME does not use task configuration avoidance #41
         for(String overlay in project.gretty.overlays)
-          dependsOn "$overlay:prepareInplaceWebAppClasses"
```

Comment:
Whitespace issue here. Not sure why it compiles without the space?! :smile:

Reply:
Groovy 🤷‍♂️